### ISSUE TITLE
Doesn't run consumer renderer if no consumerRender is passed

### ISSUE THREAD

**Issue by simonhaenisch** [ionic-team/stencil-state-tunnel#3]
I'm using this in a Stencil/Ionic PWA project, and looking at [`src/utils/state-tunnel.tsx`](https://github.com/ionic-team/stencil-state-tunnel/blob/a527126c5bbb3d4d5adca9beec84d270aac34d0d/packages/state-tunnel/src/utils/state-tunnel.tsx#L4-L11)

```tsx
function defaultConsumerRender(subscribe: SubscribeCallback<string>, renderer: Function) {
  return <context-consumer
    subscribe={subscribe}
    renderer={renderer}
  />;
}

export function createProviderConsumer<T extends object>(
  defaultState: T,
  consumerRender = defaultConsumerRender // <-- I thought this would work
) {
  // ...
}
```

I was assuming that I don't need to pass the `consumerRender` when creating my tunnel because it just uses the `defaultConsumerRender`:

```tsx
export default createProviderConsumer<{ foo: string }>({ foo: 'bar' });
```

However that doesn't work, my renderer never gets called. I need to actually pass my own `consumerRender`:

```tsx
export default createProviderConsumer<{ foo: string }>({ foo: 'bar' }, (s, r) => (
  <context-consumer subscribe={s} renderer={r} />
));
```

Not sure whether that's just an issue with my setup or because i'm importing the distributed/compiled version, and the `<context-consumer />` component just isn't able to execute the first `Consumer` child as a renderer function then?

Anyway, if you find that it's supposed to work, maybe we can get it fixed... otherwise I'd suggest to not give the `consumerRender` argument a default value (I can submit a PR for that).

Or maybe we could export `defaultConsumerRender` and then allow to do sth like

```tsx
import { createProviderConsumer, defaultConsumerRender } from '@stencil/state-tunnel';

export default createProviderConsumer<{ foo: string }>(
  { foo: 'bar' },
  defaultConsumerRender
);
```

_(not sure why that would work though)_

**Comment by jthoms1** [comment]
I believe this is an issue with the stencil component discovery process.  I am probably going to update the code to remove the default.

**Comment by jthoms1** [comment]
This is fixed by commit af900783db3bcba1ba59270201d30ec29d0602cd
